include/onnxruntime/core/platform/windows/TraceLoggingConfig.h

```diff
@@ -23,7 +23,7 @@ Module Name:
 
 // Configuration macro for use in TRACELOGGING_DEFINE_PROVIDER. The definition
 // in this file configures the provider as a normal (non-telemetry) provider.
-#ifndef ENABLE_TELEMETRY
+#ifndef TraceLoggingOptionMicrosoftTelemetry
```

Comment:
I understand this macro can also be defined somewhere else, not TraceLoggingConfigPrivate.h. But in that case are you able to ensure that header file is always included before this TraceLoggingConfig.h  file?

Reply:
It should work if it's included after (it'll override the old value)

Reply:
Though it might raise a warning for redefining the macro

Reply:
Fine. That's so tricky. :-)